src/app/new-user/new-user.page.ts

```diff
@@ -1,81 +1,89 @@
+  user = {
+    name: undefined,
+    phone: undefined,
+    email: undefined,
+    password: undefined,
+  };
 
   validationsForm: FormGroup;
   countryPhoneGroup: FormGroup;
 
   countries: Array<CountryPhone>;
 
-  constructor(private _location: Location,
-              private _router: Router,
-              private _route: ActivatedRoute,
-              private _userService: UserService,
-              private _alertService: AlertService,
-              private _challengeCodeService: ChallengeCodeService,
-              private _loadingService: LoadingService,
-              private formBuilder: FormBuilder ) {
-
-  }
+  constructor(
+    private _location: Location,
+    private _router: Router,
+    private _route: ActivatedRoute,
+    private _userService: UserService,
+    private _alertService: AlertService,
+    private _challengeCodeService: ChallengeCodeService,
+    private _loadingService: LoadingService,
+    private formBuilder: FormBuilder
+  ) {}
 
   // eslint-disable-next-line @typescript-eslint/member-ordering
   validationMessages = {
-    name: [
-      { type: 'required', message: 'Name is required.' }
-    ],
+    name: [{ type: "required", message: "Name is required." }],
     email: [
-      { type: 'required', message: 'Email is required.' },
-      { type: 'pattern', message: 'Please enter a valid email.' }
-    ],
-    password: [
-      { type: 'required', message: 'Password is required.' }
+      { type: "required", message: "Email is required." },
+      { type: "pattern", message: "Please enter a valid email." },
     ],
+    password: [{ type: "required", message: "Password is required." }],
     phone: [
-      { type: 'required', message: 'Phone is required.' },
-      { type: 'validCountryPhone', message: 'Not a valid phone number.' }
-    ]
+      { type: "required", message: "Phone is required." },
+      { type: "validCountryPhone", message: "Not a valid phone number." },
+    ],
   };
 
   ngOnInit() {
-    this.countries = [
-      new CountryPhone('US', 'United States')
-    ];
+    this.countries = [new CountryPhone("US", "United States")];
 
     const country = new FormControl(this.countries[0], Validators.required);
-    const phone = new FormControl('', Validators.compose([
-      PhoneValidator.validCountryPhone(country)
-    ]));
+    const phone = new FormControl(
+      "",
+      Validators.compose([PhoneValidator.validCountryPhone(country)])
+    );
     this.countryPhoneGroup = new FormGroup({
       country,
-      phone
+      phone,
     });
 
     this.validationsForm = this.formBuilder.group({
-      name: new FormControl('', Validators.required),
-      email: new FormControl('', Validators.compose([
-        Validators.pattern('^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+.[a-zA-Z0-9-.]+$')
-      ])),
-      password: new FormControl('', Validators.required),
-      countryPhone: this.countryPhoneGroup
+      name: new FormControl("", [Validators.required, Validators.minLength(1)]),
```

Comment:
A minLength of 1 doesn't add anything here.. the field is already required, so the user will have to type something in that field, at least one character, to satisfy that requirement. Unless you're opposed, let's remove the minLength .